extensions/amp-story/0.1/amp-story-desktop.css

```diff
@@ -82,7 +82,7 @@ amp-story[standalone][desktop] {
 }
 
 [desktop] amp-story-page {
-  transform: scale(1.0) translateX(-350%) translateY(0%) !important;
+  transform: scale(1.0) translateX(calc(-50vw - 100%)) translateY(0%) !important;
```

Comment:
LGTM, but I wonder if the precision is worth the extra computation?  Since it's offscreen, we could do something like `-200vw` and it should be okay.

Reply:
@newmuis Sounds good to me!